.github/workflows/sycl-linux-run-tests.yml

```diff
@@ -146,6 +146,13 @@ on:
           - false
           - true
 
+      skip_run:
```

Comment:
I think line 161 should be fixed instead. Maybe `{{inputs.skip_run && inputs.skip_run == 'false'}}`, or `!= true`.

Reply:
How about `if: github.event_name == 'workflow_dispatch' || inputs.skip_run == 'false'` ? Just run the E2E tests if we manually trigger the workflow.